### ISSUE TITLE
Static file finding broken on server

### ISSUE THREAD

**Issue by 2e2a** [liqd/adhocracy4#152]
To find the email logo the `finders.find('images/email_logo.png')` method is used.

This is currently broken on the servers, as the finders function does not (cannot) look into the` static/` folder created by collectstatic and the original static folders are not included in the deployment.

This can be fixed by:

1.  Always rely on the static folder created by collectstatic (also in development) and open the file there without using the finders functionality
2. Include original static folders in deployment in addition to the static folder generated by collectstatic (duplicates the static files)

**Comment by 2e2a** [comment]
@slomo, @kleingeist: What do you think?

**Comment by slomo** [comment]
Actually the deployment should include all committed files, which should also be the static folders of the app. So that would be option 2, which I also would prefer.

**Comment by slomo** [comment]
I figured out why it doesn't work anymore. We switched out project layout so that static files are now committed to `/$project_name/assets` instead of `/$project_name/static`. Since the build script only bundles all files known to git and `/static` the files can't be found by static finders on production machines any more.

Basically there are two possible solutions:
   - include `/$project_name/static` explicitly in the build script
   - move static files back to `/$project_name/static` and let webpack generate to `/static_generated` instead, which than has to be added to `STATICFILES_DIR`

The latter will result in all images and fonts being installed twice on the server.
